packages/tools/pie-css/test/scss/unit/settings/mq.spec.ts

```diff
@@ -0,0 +1,98 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import { compileCss, stripCSSWhitespace } from '../../../../utilities/compileCss';
+
+describe('settings :: mq', () => {
+    it('should render the expected breakpoint variable', () => {
+        // Arrange
+        const scssToTest = `
+          @use "sass:map";
+          @use 'settings' as *;
+
+          .foo {
+            @media (max-width: map.get($breakpoints, "sm")) {
+                color: red;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (max-width: 600px) {
+            .foo {
+              color: red;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+
+    it('should render the expected CSS media query', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('<=sm') {
+                color: red;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (max-width: 600px) {
+            .foo {
+              color: red;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+
+    it.each([
+        ['xs', '320px'],
```

Comment:
I wonder if it's possible (or desirable) to have these values somehow read from the same place as the actual definitions. If a new breakpoint were added, someone would have to remember to come and add a new test.